module/userProjects/helper.js

```diff
@@ -1560,66 +1588,66 @@ module.exports = class UserProjectsHelper {
-                //         await _entitiesInformation([data.entityId]);
-
-                //     if (!entityInformation.success) {
-                //         return resolve(entityInformation);
-                //     }
-
-                //     createProject["entityInformation"] = entityInformation.data[0];
-                //     createProject.entityId = entityInformation.data[0]._id;
-                // }
-                // if (createNewProgramAndSolution) {
-
-                //     let programAndSolutionInformation =
-                //         await this.createProgramAndSolution(
-                //             data.programId,
-                //             data.programName,
-                //             createProject.entityId ? [createProject.entityId] : "",
-                //             userToken
-                //         );
-
-                //     if (!programAndSolutionInformation.success) {
-                //         return resolve(programAndSolutionInformation);
-                //     }
-                //     createProject =
-                //         _.merge(createProject, programAndSolutionInformation.data);
-                // } 
-
-                // if (data.programId && data.programId !== "") {
-
-                //     let queryData = {};
-                //     queryData["_id"] = data.programId;
-                //     let programDetails = await programsQueries.programsDocument(queryData,
-                //             [
-                //                 "_id",
-                //                 "name",
-                //                 "description",
-                //                 "isAPrivateProgram"
-                //             ]
-                //     );
-                //     if( !programDetails.length > 0 ){
-                //         throw {
-                //             status: HTTP_STATUS_CODE['bad_request'].status,
-                //             message: CONSTANTS.apiResponses.PROGRAM_NOT_FOUND
-                //         };
-                //     } 
-                //     let programInformationData = {};
-                //     programInformationData["programInformation"] = programDetails[0];
-                //     createProject =
-                //         _.merge(createProject, programInformationData);
-                // }
+                if (data.entityId) {
+                    let entityInformation =
+                        await _entitiesInformation([data.entityId]);
+
+                    if (!entityInformation.success) {
+                        return resolve(entityInformation);
+                    }
+
+                    createProject["entityInformation"] = entityInformation.data[0];
+                    createProject.entityId = entityInformation.data[0]._id;
+                }
+                if (createNewProgramAndSolution) {
+
+                    let programAndSolutionInformation =
+                        await this.createProgramAndSolution(
```

Comment:
@Prajwal17Tunerlabs inside createProgramAndSolution this function dont call mlcore service add that api in unnati